spec/__mocks__/electron.ts

```diff
@@ -28,17 +29,24 @@ interface IPowerMonitor {
     querySystemIdleTime(): void;
 }
 
-// use config provided by test framework
 const pathToConfigDir = (): string => {
     return path.join(__dirname, '/..') as string;
 };
 
+const pathToConfigDirWindows = (): string => {
```

Comment:
do we need another method for this? Why not just have a condition in the main method to check if it's `Windows`